### DISCUSSION TITLE
GIT Commands

### DISCUSSION THREAD

**Discussion by IT21322662** [community/community#70615] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need to know the git commands

**Comment by Dineth-H** [comment]
1. Clone a Repository:

To copy a Git repository to your computer, use this command, replacing <repository_url> with the repository's URL:
`git clone <repository_url>`

2.. Commit Changes:

After making changes, follow these two steps:
  Add your changes:
  `git add .`

  Commit your changes with a message:
  `git commit -m "Your commit message here"`

3. Merge Branches:

If you have multiple branches and want to combine changes, use these commands:

  Switch to the target branch (e.g., main):
  `git checkout main`

  Merge changes from a feature branch into the main branch:
  `git merge feature-branch`

4. Create a Pull Request:
When collaborating on Git platforms like GitHub or GitLab, propose changes with these steps:

  Push your changes to your remote repository:
  `git push origin feature-branch`

  Create a pull request, but the command may vary by platform. Here's an example for GitHub using gh:
  `gh pr create --base main --head feature-branch --title "Your pull request title" --body "Your pull request description"`

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
